hadoop-hdds/docs/content/feature/SCM-HA.md

```diff
@@ -33,14 +33,8 @@ This document explains the HA setup of Storage Container Manager (SCM), please c
 
 ## Configuration
 
-HA mode of Storage Container Manager can be enabled with the following settings in `ozone-site.xml`:
+As of Apache Ozone 2.0, SCM HA is always enabled. Test deployments can continue to use single node Ratis for SCM if needed.
```

Comment:
The user might get confused with SCM HA and Ratis if they don't have understanding on how HA is implemented.
We can remove this line completely as the end user doesn't have to know about Ratis to run single or multiple (HA) SCM.